src/style.css

```diff
@@ -0,0 +1,104 @@
+@import url('https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css');
+@import url('https://fonts.googleapis.com/css2?family=Dosis:wght@300&family=Nerko+One&display=swap');
+
+body {
+  font-family: 'Dosis', sans-serif;
+  font-style: normal;
+  margin: 0;
+}
+
+main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.list {
+  width: 60%;
+  margin-top: 100px;
+}
```

Comment:
## [ Optional Change ] 
You did well on your styles but you could add a little more style to make things look elegant and match the original design by giving the todo an elevated box shadow.

- [ ] I highly recommend you work on this change to improve the look of your app by adding this style to your `.list` class `box-shadow: 1px 4px 30px rgba(0, 0, 0, 0.1);`. You're welcome 👍 



**See the original design**
![image](https://user-images.githubusercontent.com/50941074/206130964-12051602-031e-4476-bc62-8ee2870c5579.png)

Reply:
Thanks 😊,  for approval, I will work on optional changes.